packages/synapse-react-client/src/synapse-queries/entity/useEntity.ts

```diff
@@ -46,21 +49,32 @@ import { getNextPageParamForPaginatedResults } from '../InfiniteQueryUtils'
 import { KeyFactory } from '../KeyFactory'
 import { useGetEntityBundleQueryOptions } from './useEntityBundle'
 
+export function useGetEntityQueryOptions<T extends Entity>() {
```

Comment:
Export a way to get the query options so we can pass it into `queryClient` imperatively, e.g. 

```js
const options = getEntityQueryOptions(args)
queryClient.fetchQuery(options)
```